pytorch_to_returnn/import_wrapper/wrap.py

```diff
@@ -77,6 +81,8 @@ def unwrap(obj):
 def _nested_transform(obj, transform):
   if type(obj) == tuple:
     return tuple([transform(item) for item in obj])
+  if issubclass(type(obj), tuple) and type(obj) is not tuple:
+    return type(obj)(*(transform(item) for item in obj))
```

Comment:
I suppose so, I'll remove it.

Reply:
Another thing, which should maybe be at the top of this function:
```
if isinstance(obj, WrappedObject):
  return obj
```
Otherwise it would keep the wrapped container type (namedtuple, `PackedSequence`) but unwrap all its content (the tensors), and then it might be confused in other parts? Not sure... But it should also not cause problems, and be safe, and better to have this.